packages/server/graphql/public/mutations/removeMultipleOrgUsers.ts

```diff
@@ -113,7 +125,7 @@ const removeMultipleOrgUsers: MutationResolvers['removeMultipleOrgUsers'] = asyn
         publish(
           SubscriptionChannel.TASK,
```

Comment:
-1 now that I've read through the updater, I think all team members, remaining and kicked out need to get this update. We also should do this once per team and not {user, team} tuple.